TodoListApp/TodoList.Api.Tests/DummyTests.cs

```diff
@@ -0,0 +1,14 @@
+﻿using NUnit.Framework;
+
+namespace TodoList.Api.Tests
+{
+    [TestFixture]
+    public class DummyTests
+    {
+        [Test]
+        public void BasicMathTest()
+        {
+            Assert.AreEqual(5, 2 + 3);
```

Comment:
An `Assert.True` would do as well, but this one is OK nonetheless 😃

Reply:
👍